execution/execution.py

```diff
@@ -407,7 +417,7 @@ def swatch_list():
     return response.json()
 
 
-def swatch_list_all():
+def swatch_list_all(current_only=False):
```

Comment:
Hello @starnold-redhat 
Question here, do you believe it to be interesting to add this feature on swatch as well? I'm asking that because the query o swatch endpoint it's pretty quick, independent of the # of entries. Also, this will not match the # as we see on HBI, in an env for example with thousands of entries on HBI, swatch will contain a few hundred and it will be a single query here (on HBI, for a full dump, we need to do 2 calls at this moment).

Please, let me know what you think.
Waldirio

Reply:
Hi,

I think so.  As you know, the hcs-collector uses the swatch info for determining rhel subs.  And the swatch info still contains systems that havent checked in for a while.   So it makes sense to me to remove these systems that havent checked in from both inventory and swatch.  

The check won't slow down the call at all - it just doesnt save some of the entries.

Do you have a concern ?

Steve

Reply:
Hi,

Not at all. Testing the code atm.

Thank you!
Waldirio